🐞 Bug: linuxheaders fallback fails to build at `make defconfig`

**Describe the bug**
I'm running into issues installing `linuxheaders`

**To Reproduce**
Steps to reproduce the behavior:

```
$ crew install linuxheaders

linuxheaders: Linux headers for Chrome OS.
https://kernel.org/
Version: 5.10
License: GPL-2
Performing pre-flight checks...
Precompiled binary available, downloading...
Linuxheaders archive downloaded.
Unpacking archive using 'tar', this may take a while...
Performing pre-install...
Performing install...
Performing post-install for linuxheaders...
The installed kernel headers do NOT match the current kernel version.
Please build and install the appropriate kernel headers with:
crew reinstall -s linuxheaders
Linuxheaders installed!
```
(I see that this warning was added recently in #7673 )
```
$ crew reinstall -s linuxheaders

linuxheaders: Linux headers for Chrome OS.
Performing pre-flight checks...
Downloading source...
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint:   git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint:   git branch -m <name>
Initialized empty Git repository in /usr/local/tmp/crew/linuxheaders.20221219014001.dir/.git/
remote: Finding sources: 100% (78176/78176)
remote: Total 78176 (delta 4362), reused 21920 (delta 4362)
Receiving objects: 100% (78176/78176), 214.91 MiB | 3.59 MiB/s, done.
Resolving deltas: 100% (4362/4362), done.
From https://chromium.googlesource.com/chromiumos/third_party/kernel
 * branch                chromeos-5.10 -> FETCH_HEAD
 * [new branch]          chromeos-5.10 -> origin/chromeos-5.10
Updating files: 100% (73601/73601), done.
HEAD is now at 9e7a457ad BACKPORT: FROMGIT: drm/amd/display: Waiting for 1 frame to fix the flash issue on PSR1
Repository downloaded.
Building from source, this may take a while...
Preconfiguring package...
Makefile:600: arch/aarch64/Makefile: No such file or directory
make: *** No rule to make target 'arch/aarch64/Makefile'.  Stop.
/usr/local/lib/crew/lib/package.rb:306:in `system': Command failed with exit 2: bash
linuxheaders failed to install: `CREW_DISABLE_ENV_OPTIONS="1" bash -c PATH=/usr/local/tmp/crew/linuxheaders.20221219014001.dir/crew_bin:$PATH make defconfig` exited with 2
```

**Expected behavior**
Correct `linuxheaders` for kernel version is installed

**Environment: Paste the output of `crew sysinfo -v` between the two `***` below**

***
<details><summary>Expand</summary>

- Architecture: `aarch64` (`aarch64`)
- Processor vendor: `ARM`
- User space: `32-bit`
- Chromebrew Kernel version: `5.10`
- Chromebrew Running in Container: `false`

- Chromebrew version: `1.29.0`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib`

- Last update in local repository: 8760ec6 `Codium 1.74.0.22342 => 1.74.1.22349 (#7750) (35 hours ago)`

- OS variant: `Chrome OS`
- OS version: `kukui-release/R108-15183.69.0`
- OS channel: `stable-channel`

</details>
***

**Additional context**
```
$ uname -a

Linux localhost 5.10.147-20158-g0a75306e5639 #1 SMP PREEMPT Wed Dec 7 23:51:02 PST 2022 aarch64 GNU/Linux
```

--- a/packages/linuxheaders.rb
+++ b/packages/linuxheaders.rb
@@ -78,8 +78,14 @@ class Linuxheaders < Package
     FileUtils.mkdir_p('crew_bin')
     @workdir = Dir.pwd
     FileUtils.ln_sf "#{CREW_PREFIX}/bin/ld.bfd", "#{@workdir}/crew_bin/ld"
-    system "PATH=#{@workdir}/crew_bin:$PATH make defconfig"
-    system "PATH=#{@workdir}/crew_bin:$PATH make headers_install INSTALL_HDR_PATH=#{CREW_DEST_PREFIX}"
+    @kbuild_arch = case ARCH
+                   when 'armv7l', 'aarch64'
+                     'arm'
+                   else
+                     'x86'
+                   end
+    system "ARCH=#{@kbuild_arch} PATH=#{@workdir}/crew_bin:$PATH make defconfig"
+    system "ARCH=#{@kbuild_arch} PATH=#{@workdir}/crew_bin:$PATH make headers_install INSTALL_HDR_PATH=#{CREW_DEST_PREFIX}"
     Dir.chdir("#{CREW_DEST_PREFIX}/include") do
       system "for file in $(find . -not -type d -name '.*'); do
                 rm ${file};

